backends/tfhe-cuda-backend/cuda/src/fft128/fft128.cuh

```diff
@@ -287,6 +287,25 @@ batch_convert_u128_to_f128_as_torus(double *out_re_hi, double *out_re_lo,
       &in[blockIdx.x * params::degree + params::degree / 2]);
 }
 
+// params is expected to be full degree not half degree
+template <class params>
+__global__ void
+batch_convert_u128_to_f128_strided_as_torus(double *d_out,
```

Comment:
Why is there strided in the name?

Reply:
there are two versions of batch convert:
1) `[re_hi_0, re_hi_1, ... re_hi_n, re_lo_0, re_lo_1, ... re_lo_n, im_hi_0, im_hi_1, ..., im_hi_n,  im_lo_0, im_lo_1, ..., im_lo_n]`
2) strided one: `[re_hi_0, re_lo_0, im_hi_0, im_lo_0, re_hi_1, re_lo_1, im_hi_1, im_lo_1, ...,re_hi_n, re_lo_n, im_hi_n, im_lo_n, ]`

I need to add comments there.

Reply:
Aaah ok! Hmm do you observe that one is more efficient than the other? Do we need to keep both?

Reply:
From our discussion the idea is to drop the coalesced version, we would add it back if needed if we do the 128 bit multi-bit PBS at some point. Not sure we'll do it because key sizes will become an issue.